### ISSUE TITLE
Amethyst not working in 10.15.4

### ISSUE THREAD

**Issue by mbluemer** [ianyh/Amethyst#1023]
**Describe the bug**
Installed Amethyst and gave it accessibility access in System Preferences but it's failing to capture the mod1 or mod2 key presses. Instead other system functionality is executed such as typing special characters or take a window screenshot.

**Versions:**
 - macOS: 10.15.4
 - Amethyst: 0.15.3
 
**Debug Info**
```
Version: 0.15.3 (88)

OS version: Version 10.15.4 (Build 19E287)

Has permissions: false

Screens:
	(0.0, 0.0, 1792.0, 1120.0) [(0.0, 0.0, 1792.0, 1120.0)]

Manageable applications:
	ViewBridgeAuxiliary (com.apple.ViewBridgeAuxiliary)
	Firefox Developer Edition (org.mozilla.firefoxdeveloperedition)
	Amazon Chime (com.amazon.Amazon-Chime)
	Control Strip (com.apple.controlstrip)
	Cisco AnyConnect Secure Mobility Client (com.cisco.Cisco-AnyConnect-Secure-Mobility-Client)
	Slack (com.tinyspeck.slackmacgap)
	Notes (com.apple.Notes)
	talagent (com.apple.talagent)
	ViewBridgeAuxiliary (com.apple.ViewBridgeAuxiliary)
	AppSSOAgent (com.apple.AppSSOAgent)
	QuickLookUIService (PID 897) (com.apple.quicklook.QuickLookUIService)
	Finder (com.apple.finder)
	EmojiFunctionRowIM (com.apple.inputmethod.EmojiFunctionRowItem)
	Spotify (com.spotify.client)
	studentd (com.apple.studentd)
	coreautha (com.apple.LocalAuthentication.UIAgent)
	Alacritty (io.alacritty)
	storeuid (com.apple.storeuid)

Configuration:
mod2: (
    option,
    shift,
    control
)
layouts: (
    tall,
    wide,
    fullscreen,
    column
)
floating: (
)
screen-padding-top: 25
mod1: (
    option,
    shift
)
window-margins: 1
screen-padding-bottom: 0
use-canary-build: 0
debug-layout-info: 0
window-resize-step: 5
screen-padding-right: 0
focus-follows-mouse: 0
restore-layouts-on-launch: 1
floating-is-blacklist: 1
ignore-menu-bar: 0
screen-padding-left: 0
mouse-follows-focus: 0
window-minimum-width: 0
window-margin-size: 10
new-windows-to-main: 0
follow-space-thrown-windows: 1
enables-layout-hud-on-space-change: 1
window-minimum-height: 0
float-small-windows: 1
enables-layout-hud: 1
```

The `Has permissions: false` is kind of weird given that I've provided the proper permissions in System Preferences.

**Comment by ianyh** [comment]
I probably need to drop support for 10.15 unrelated to this.
